app/services/create_verification.rb

```diff
@@ -8,6 +8,11 @@ def initialize(statement:, params:)
   def run
     statement.verifications.create!(params)
 
+    # Find duplicate statements and update their verifications
+    statement.duplicate_statements.each do |duplicate_statement|
+      duplicate_statement.verifications.create!(params)
```

Comment:
I was wondering if it's worth only doing this if the most recent verification of the duplicate statement has a different verification status, so that it's not unnecessarily creating verification objects. But then again, since it might be a different user triggering this later action, perhaps it's of some interest to see that the statement would have been re-verified by the action of someone else later because it's a duplicate.